portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,8 +13,9 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
 import java.io.IOException;
+import com.google.gson.Gson;
+import java.util.ArrayList;
```

Comment:
java style nit: imports should be in alphabetical order (does this auto-formatter not fix this for you?)